api/routes/webtoken.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"fmt"
+	"log"
+
+	"github.com/shellhub-io/shellhub/api/apicontext"
+	"github.com/shellhub-io/shellhub/api/nsadm"
+	"golang.org/x/crypto/bcrypt"
+)
+
+const (
+	GetTokenListURL = "/token"
+)
+
+func GenerateToken(c apicontext.Context) string {
+	hash, err := bcrypt.GenerateFromPassword([]byte(c.Tenant()), bcrypt.DefaultCost)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Here we ought to return the error.

Reply:
This doesn't fix the comment. We ought to use `(string, error)` as the return type.